build-image/openhabian.conf

```diff
@@ -23,8 +23,12 @@ wifi_psk=""
 # See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 or /usr/share/zoneinfo/zone.tab
 wifi_country=""
 
-# vim: filetype=sh
-
 # Java architecture mode
 # Use "32-bit" or "64-bit"
-java_arch="64-bit"
\ No newline at end of file
+java_arch="64-bit"
+
+# repo to clone from (if not master for build and test)
+repositoryurl=https://github.com/openhab/openhabian.git
+clonebranch=master
```

Comment:
I know it's messy but for a start, let not build-image/openhabian*conf get out of sync with openhabian.conf.dist
See #683 is adding a "mode" parameter
(check https://github.com/mstormi/openhabian/blob/buildsystem-init.bash/build-image/first-boot.bash)

In general, it's probably a better idea to refactor this and _generate_ build-image/openhabian*conf _from_ openhabian.conf.dist, or turn them into 'diff' files and copy openhabian.conf.dist, then apply the diff everywhere in the code where that so far accessed static build-image/openhabian*conf.

Reply:
I would propose to keep them in sync manually for now. Let us first focus on the important problems, then remove pinea64 support from build.bash and then see what to refactor....
Ok?